### PR TITLE
[NEW] Wear Module

### PR DESCRIPTION
**Pull request by aniketsingh03**
@RocketChat/android

This pr is specific to GSoC-2018 project.
Initialisation of the wear app. The wear (android watch) app for Rocket Chat will be available soon. This is the first pull request in its making.

### CODE REVIEW

**Comment by philipbrito** [comment]
Why not using our app icon on the wear app?

File: wear/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<resources>
+    <string name="app_name">Wear</string>
```
Comment (philipbrito) [inline wear/src/main/res/values/strings.xml]: Wear? It should be `Rocket.Chat` (like the Android app name).

File: wear/build.gradle [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+android {
+    compileSdkVersion 26
+
+
+    defaultConfig {
+        applicationId "chat.rocket.android.wear"
+        minSdkVersion 23
+        targetSdkVersion versions.targetSdk
+        versionCode 1
+        versionName "1.0"
```
Comment (philipbrito) [inline wear/build.gradle]: Change to `1.0.0` (semantic versioning).

File: wear/build.gradle [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation libraries.kotlin
+    implementation 'com.google.android.support:wearable:2.3.0'
+    implementation 'com.google.android.gms:play-services-wearable:15.0.1'
+    implementation 'com.android.support:percent:26.1.0'
+    implementation 'com.android.support:support-v4:26.1.0'
```
Comment (philipbrito) [inline wear/build.gradle]: You can declare it as the same way we declare for the Android app. Change it please!

File: wear/build.gradle [hunk 4]
```diff
@@ -0,0 +1,38 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+android {
+    compileSdkVersion 26
+
+
```
Comment (philipbrito) [inline wear/build.gradle]: Remove this blank space.

**Review by philipbrito — CHANGES_REQUESTED** [review]

**Comment by aniketsingh03** [comment]
@filipedelimabrito I'll just resolve all these issues and push in some moments.

**Comment by aniketsingh03** [comment]
@filipedelimabrito Are we planning to keep this from merging until we have some features in the app?

File: wear/build.gradle [hunk 5]
```diff
@@ -0,0 +1,37 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+android {
+    compileSdkVersion 26
+
```
Comment (philipbrito) [inline wear/build.gradle]: Missing `buildToolsVersion versions.buildTools` here! 
In that way we are unable to launch the project.

**Review by philipbrito — CHANGES_REQUESTED** [review]

**Review by philipbrito — APPROVED** [review]
Thanks @aniketsingh03!

**Pull request merged**